makefiles/Makefile.docs-mongoid-railsmdb

```diff
@@ -0,0 +1,21 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+
+ifeq ($(STAGING_USERNAME),)
+	USER=$(shell whoami)
+else
+	USER=$(STAGING_USERNAME)
+endif
+
+PROJECT=mongoid-railsmdb
+MUT_PREFIX ?= $(PROJECT)
+
+REPO_DIR=$(shell pwd)
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+
+get-build-dependencies: 
+	echo "Published branches information stored in Atlas collection"
```

Comment:
It's often idiomatic to have commands which are just an `echo` have the `@` sigil to suppress printing the command, then running it, and kind of getting the same thing printed twice

Reply:
Makes sense. I've gone ahead and added the suppression